pageserver/src/tenant/mgr.rs

```diff
@@ -247,24 +234,25 @@ impl BackgroundPurges {
     /// Although we are cleaning up the tenant, this task is not meant to be bound by the lifetime of the tenant in memory.
     /// Thus the [`BackgroundPurges`] type to keep track of these tasks.
     pub fn spawn(&self, tmp_path: Utf8PathBuf) {
-        let mut guard = self.0.lock().unwrap();
-        let jset = match &mut *guard {
-            BackgroundPurgesInner::Open(ref mut jset) => jset,
-            BackgroundPurgesInner::ShuttingDown(_) => {
-                warn!("trying to spawn background purge during shutdown, ignoring");
-                return;
+        if self.0.is_closed() {
+            warn!(
+                ?tmp_path,
```

Comment:
this can use display because it's utf8, no?

Reply:
ah yes just cutpasted it.

Reply:
[89cb5f9](https://github.com/neondatabase/neon/pull/8650/commits/89cb5f9610f2ac480cb2662fcedd9df5aabe440b)